article/views.py

```diff
@@ -0,0 +1,64 @@
+from django.shortcuts import render
+from rest_framework.response import Response 
+from rest_framework.decorators import api_view
+from rest_framework import status
+from .models import Article , ArticleComment , ArticleVote
+from .serializers import ArticleSerializer , ArticleCommentSerializer
+# Create your views here.
+
+@api_view(['GET'])
+def artciles(request):
```

Comment:
change function name to get_articles

Reply:
we are following camel case typing ! so in other views its done the same way i did .
so we have to keep it this way 
see - feed/views.py 

`@api_view(['POST'])
def updateVote(request):
    user = request.user 
    data = request.data`